src/Init/Data/Int/Basic.lean

```diff
@@ -28,32 +55,69 @@ instance : OfNat Int n where
 namespace Int
 instance : Inhabited Int := ⟨ofNat 0⟩
 
+/-- Negation of a natural number. -/
 def negOfNat : Nat → Int
   | 0      => 0
   | succ m => negSucc m
 
 set_option bootstrap.genMatcherCode false in
+/-- Negation of an integer.
+
+  Implemented by efficient native code. -/
 @[extern "lean_int_neg"]
 protected def neg (n : @& Int) : Int :=
   match n with
   | ofNat n   => negOfNat n
   | negSucc n => succ n
 
+/--
+  The `Neg Int` default instance must have priority higher than `low`
+  since the default instance `OfNat Nat n` has `low` priority.
```

Comment:
this should be a regular comment, not a doc comment. Doc comments on instances are mostly useless to begin with